common/xml.go

```diff
@@ -0,0 +1,133 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+const (
```

Comment:
nit: these constant names can be a bit shorter:
- `openParen`
- `closeParen`
- `closeParenWithSlash`